dynamax/linear_gaussian_ssm/models.py

```diff
@@ -56,26 +68,37 @@ class LinearGaussianSSM(SSM):
     The parameters of the model are stored in a :class:`ParamsLGSSM`.
     You can create the parameters manually, or by calling :meth:`initialize`.
 
+    Please note that we adopt the convention of Murphy, K. P. (2022), "Probabilistic machine learning: Advanced topics",
+    rather than Särkkä, S. (2013), "Bayesian Filtering and Smoothing" for indexing parameters of LGSSM, where we start 
+    initial index at 0 instead of 1, which is not exactly in line with the former book. This tends to be a source of
+    confusion sometimes. As such, $F_0$, $B_0$, $b_0$, $Q_0$ are always ignored and the prior specified by $m$ and $S$
+    is used as the distribution of the initial state.
+
```

Comment:
There are several conflicts of indexing style. I added this note here to make sure that this code base follows the notation of Murphy (2023), which is $z_t \sim \mathcal{N}(F_t z_{t - 1} + B_t u_t + b_t, Q_t)$.

By the way, I personally prefer Sarkka's style $z_{t+1} \sim \mathcal{N}(F_t z_t + B_t u_t + b_t, Q_t)$, and I believe it makes the codebase look a little simpler, and I don't mind modifying the whole codebase of LGSSM to adopt his indexing if the code owners prefer it.